sdk/logs/logger.go

```diff
@@ -40,7 +40,10 @@ func (l logger) Emit(logRecord logs.LogRecord) {
 		return
 	}
 
-	pr := l.provider.resource
+	pr, err := resource.Merge(l.provider.resource, logRecord.Resource())
```

Comment:
Thanks for the suggestion.

That sound more to be a [Scope Resource](https://opentelemetry.io/docs/specs/otel/glossary/#instrumentation-scope) and we have it defined in [proto](https://github.com/open-telemetry/opentelemetry-proto-go/blob/main/otlp/logs/v1/logs.pb.go#L343) and [model](https://opentelemetry.io/docs/specs/otel/logs/data-model/#log-and-event-record-definition). 
and correct implementation should be 

```
pr, err := logRecord.Resource()
```

But from this point provider resource lost and should be taken in `LogRecordProcessor` implementation. However, 
we don't have another way yet to access [LoggerProvider](https://github.com/agoda-com/opentelemetry-logs-go/blob/master/sdk/logs/provider.go#L105) resources in the [Exporter](https://github.com/agoda-com/opentelemetry-logs-go/blob/master/exporters/otlp/otlplogs/exporter.go#L77)

I will merge that MR to have both resources merged as currently logs is not getting grouped by resource/scope when sent to OTLP, but this should be refactored in the future.